core/src/main/java/uk/gov/justice/services/core/handler/HandlerMethod.java

```diff
@@ -85,8 +92,15 @@ private static boolean isEnvelope(final Class<?> clazz) {
      */
     @SuppressWarnings("unchecked")
     public Object execute(final JsonEnvelope envelope) {
+        trace(LOGGER, () -> format("Dispatching to handler %s.%s: %s",
+                handlerMethod.getClass().getSimpleName(), handlerMethod.getName(), toEnvelopeTraceString(envelope)));
         try {
-            return handlerMethod.invoke(handlerInstance, envelope);
+            final Object obj = handlerMethod.invoke(handlerInstance, envelope);
+            trace(LOGGER, () -> format("Response from handler %s.%s: %s",
```

Comment:
This should be printing the response (obj, cast into a JsonEnvelope), but instead its printing the request